tests/Rector/Architecture/Factory/NewObjectToFactoryCreateRector/config.yml

```diff
@@ -0,0 +1,6 @@
+services:
+  Rector\Rector\Architecture\Factory\NewObjectToFactoryCreateRector:
+    arguments:
```

Comment:
@TomasVotruba well, this works, but please how would the config look like, if i wanted to use more objects and more factories, because `arguments` key is unique for this service. Sorry for struggling with this basic symfony/yaml feature

Reply:
That's part of learning Symfony.

It took me while to find this feature, it's here: https://symfony.com/blog/new-in-symfony-3-3-simpler-service-configuration#short-syntax-for-service-configuration